src/extension.ts

```diff
@@ -100,10 +98,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 				}).catch( (err) => {
 					console.log(err);
 				});
-				await removeOutputChannelForIntegrationViaKubectl(integrationName)
-				.catch( (err) => {
-					console.log(err);
-				});
+				// TODO: we need to look into closing the log view when the integration is stopped
```

Comment:
have you created a specific issue yet?

Reply:
https://github.com/camel-tooling/vscode-camelk/issues/310

Reply:
I also created https://github.com/camel-tooling/vscode-camelk/issues/311 to track the issue of the stale log views when updating an integration changes the pod.